conda/recipes/cuml-cpu/meta.yaml

```diff
@@ -11,7 +11,7 @@ package:
   version: {{ version }}
 
 source:
-  git_url: ../../..
+  path: ../../..
```

Comment:
How does this change relate to the context here?

Reply:
If conda recipes specify the source as a `git_url`, even if a path is provided all local changes (definitely unstaged changes, but I believe even staged but uncommitted changes) are discarded during the conda build. Since the other changes in this PR involve modifying the source before the build, changing `git_url` to `path` is necessary.

[We've been wanting to make the change anyway](https://github.com/rapidsai/ops/issues/2537), so this PR just provides an extra impetus.